docs/reference/modules/global.md

```diff
@@ -0,0 +1,69 @@
+---
+extends: '@apostrophecms/piece-type'
+---
+
+# `@apostrophecms/global`
+
+**Alias:** `apos.global`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module populates the `req.data.global` with content or settings that are utilized site-wide. It is configured through "implicit sub-classing" of the schema fields present in the parent module. New fields are added and grouped identically to any widget or pieces module. The data from these fields is available to templates as `data.global`.
+
+::: note
+There can only be *one* global doc.
+:::
+
+## Options
+|  Property | Type | Description |
+|---|---|---|
+|`deferWidgetLoading` | Boolean | If set to `true`, any widget with the option [`deferred: true`](https://v3.docs.apostrophecms.org/reference/modules/widget-type.html#deferred) will not be loaded until the end of loading the global doc. |
+
+### `deferWidgetLoading`
+
+If set to `true`, the `deferWidgetLoading` option reduces the number of queries required when loading the global doc. Any widget with the option `deferred: true` used on the page will not be loaded until the entire global doc is loaded.
+
+#### Example
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+ options: {
+   deferWidgetLoad: true
+ },
+ fields: {
+   add: {
+     githubUrl: {
+       type: 'url',
+       label: 'Organization GitHub url'
+     },
+     footerMissionStatement: {
+       type: 'area',
+       label: 'Footer mission statement',
+       options: {
+         widgets: {
+           '@apostrophecms/rich-text': {}
+         }
+         // Other fields with multiple widget types
```

Comment:
This comment is confusing. I think it's clear you can add more than one field, more than one widget type, etc. to someone who's already been through the basics of the guide, which should be a safe assumption in the reference docs.

Reply:
Extra comment eliminated.